src/Writer/StandaloneExtensionManager.php

```diff
@@ -9,6 +9,8 @@
 
 namespace Zend\Feed\Writer;
 
+use Zend\ServiceManager\Exception\InvalidServiceException;
```

Comment:
You cannot use ServiceManager exceptions in here. Use `Zend\Feed\Writer\Exception\InvalidArgumentException`